### PR TITLE
NOISSUE - Update event fields naming

### PR DESCRIPTION
**Pull request by dborovcanin**
### What does this do?
This pull request updates fields names in the Bootstrap service events.

### Have you included tests for your changes?
Yes, I've updated tests.

### Did you document any new/modified functionality?
No, event sourcing documentation will be updated in #698.

### CODE REVIEW

File: bootstrap/redis/producer/events.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ type createConfigEvent struct {
 
 func (cce createConfigEvent) encode() map[string]interface{} {
 	return map[string]interface{}{
-		"id":          cce.mfThing,
+		"thing_id":    cce.mfThing,
```
Comment (drasko) [inline bootstrap/redis/producer/events.go:54]: Why do we use snake_case here? Isn't cammelCase something that is Go standard?

**Review by drasko — CHANGES_REQUESTED** [review]

File: bootstrap/redis/producer/events.go (hunk 1 above)
Comment (anovakovic01) [reply bootstrap/redis/producer/events.go:54]: This is not `go` event but Redis event. If events were in JSON format, we would use underscore (AFAIK we already use it for JSON in our API).

Comment (dborovcanin) [reply bootstrap/redis/producer/events.go:54]: I've changed `id` to `thing_id` just to be in sync with Things service. I think it's also more precise. There is no other reason for this change.

**Review by drasko — DISMISSED** [review]
LGTM

**Review by nmarcetic — DISMISSED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**